terraform/environments/data-platform/s3.tf

```diff
@@ -0,0 +1,103 @@
+module "s3-bucket" { #tfsec:ignore:aws-s3-enable-versioning
+  source = "github.com/ministryofjustice/modernisation-platform-terraform-s3-bucket?ref=v6.3.0"
+
+  bucket_prefix = "data-platform-landing"
```

Comment:
Can you pls change this to data-platform-sandbox or something. As i explained there will be only one bucket and a landing zone / Curated zone will be there